packages/core/test/interpreter.test.ts

```diff
@@ -1331,7 +1257,7 @@ describe('interpreter', () => {
     });
   });
 
-  describe('off()', () => {
+  describe('.off()', () => {
```

Comment:
this doesn't exist anymore so this describe block should be called something else